### PR TITLE
Fix readme and solution for building custom immage [WIP]

### PR DESCRIPTION
**Pull request by holdenk**
Incase @ifilonenko wants to take over while I'm off dealing with lasers and other "fun" things.

### CODE REVIEW

File: multi-cloud/README.md [hunk 1]
```diff
@@ -400,40 +400,68 @@ it out pretty easily on your own once this is done.
...
+
+Since we're deploying this on a cluster we'll need to put the image in something called a container registry.
+To give your local Docker permission to your the [Google container registry](https://cloud.google.com/container-registry/) is configured with:
+
+```bash
+gcloud auth configure-docker
+```
+
+
+Your Google container registry](https://cloud.google.com/container-registry/) is at `gcr.io/${GOOGLE_PROJECT}`, so we'll tag our image to this base and push it up:
```
Comment (ifilonenko) [inline multi-cloud/README.md]: `GOOGLE_CLOUD_PROJECT`

File: multi-cloud/fast-start.sh [hunk 2]
```diff
@@ -71,6 +71,8 @@ if ! GOOGLE_PROJECT=$(gcloud config get-value project 2>/dev/null) ||
   echo "gcloud config set project $latest_project" >> ~/.bashrc
 fi
 GOOGLE_PROJECT=$(gcloud config get-value project 2>/dev/null)
+echo "export GOOGLE_PROJECT=$GOOGLE_PROJECT" >> ~/.bashrc
```
Comment (ifilonenko) [inline multi-cloud/fast-start.sh]: This should already be set to be `GOOGLE_CLOUD_PROJECT.` I guess this would be dummy proof?

**Comment by holdenk** [comment]
So, there's no reference to the env variable `GOOGLE_CLOUD_PROJECT` so I'm sticking with the one that we used before. changing it doesn't give us anything I can see?

**Comment by holdenk** [comment]
oh wait as per discussions: that isn't set if the person hasn't set the project in the WebUI so we don't want to depend on that.

**Pull request merged**